### DISCUSSION TITLE
v3.16.0

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#10598] [Releases & Announcements]
## Changelog

- Design update for report pages
- Design update for contact pages
- Updated audio notification settings with new sounds
- Ability to lock conversations to single thread in API channels
- Captain copilot integration to sidebar (alpha)
- Ability to create conversation directly from email / phone number 
- Add support for Arcade videos on articles
- Other Components and Design changes in preparation for v4
- Numerous bug fixes and enhancements

Thanks to @honzasterba, @giquieu, @nguyentranchung for the contributions

<hr /><em>This discussion was created from the release <a href='https://github.com/chatwoot/chatwoot/releases/tag/v3.16.0'>v3.16.0</a>.</em>

**Comment by AhmedSoubhi** [comment]
I'm getting this error when I clone the GitHub reposit to my vs code so i can modify it I'm getting this error exec docker/entrypoints/rails.sh: no such file or directory while I'm preparing the database  docker compose run --rm rails bundle exec rails db:chatwoot_prepare. I have made sure all the directories are in the same place but I'm still getting this error please help me fix it !

**Comment by glauberssoutodossantos** [comment]
hi team....

@sojan-official `Captain copilot integration to sidebar (alpha)` how this feature will works?

**Reply by sojan-official** [reply]
We are rolling this out to select customers on Chatwoot cloud at the moment who has access to captain.  We will follow up with details on general release.

**Comment by YulsMD** [comment]
Hi, when you're in the _Contacts_ section and try to send a message using saved templates, the options don't show with the / command. Inside a conversation, it works fine. It seems **templates aren't being read in the _Contacts_ section.**
<img width="679" alt="Screenshot 2025-01-07 at 17 29 16" src="https://github.com/user-attachments/assets/af9d6424-4e81-44c7-9b9c-366a8023b71a" />

**Comment by Faizan-Ahmad04** [comment]
I've update chatwoot from 13.13.0 to 13.16.0 that introduce vite now when i open my widget script in CodePen locally the widget is not loaded and give error 
 
Started GET "/packs/js/sdk.js" for ::1 at 2025-03-24 16:29:53 +0530
  
ActionController::RoutingError (No route matches [GET] "/packs/js/sdk.js"):
 
Could you please let me know what is the issue here
